database/realm/RealmDatabaseManager.py

```diff
@@ -21,6 +21,29 @@
 
 
 class RealmDatabaseManager(object):
+    # Realm stuff
+    
+    @staticmethod
+    def realm_get_name(id=1):
```

Comment:
It would be better to directly return the whole Realmlist object, instead of doing 3 different queries for each field.